biscuit-auth/src/datalog/expression.rs

```diff
@@ -65,6 +65,9 @@ pub enum Binary {
     Sub,
     Mul,
     Div,
+    BitwiseAnd,
+    BitwiseOr,
+    BitwiseXor,
```

Comment:
out of consistency with the protobuf and the parser definitions, these could be put at the end of the list

Reply:
:bow: